doc/en/components/data-grid-row-paging.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: $Platform$ Data Grid | Row Paging | Infragistics
+title: $Platform$ grid/table pagination – $ProductName$ 
```

Comment:
use camel case and there should be "Infragistics" instead of $ProductName$ 

Reply:
Alexander wants React grid/table pagination – Ignite UI for React 

Reply:
Why not have it React Grid/Table Pagination | Ignite UI for React | Infragistics

Reply:
all topics do not include $ProductName$ so what is special about this topic to include $ProductName$ or is he saying that all topics should include it?

Reply:
will update rest of topics later